### PR TITLE
Introduction of a dat.GUI GUI configurator.

### PR DESCRIPTION
**Pull request by EricBoix**
Help, Temporal and About are now classes that integrate
their html and are associated with their respective css.

dat.GUI controls are hooked up with internal variables of
those classes. data.GUI thus doesn't need any proper variables
(that would need to be kept in sync).

### CODE REVIEW

File: UDV-Core/examples/Demo.js [hunk 1]
```diff
@@ -56,20 +56,53 @@ const optionsEditMode= {
...
+aboutController.onFinishChange( function(value) { about.refresh(); } );
+
+// About subwindow
+helpController = datDotGUI.add( help, 'windowIsActive'
+                              ).name( "Help" ).listen();
+helpController.onFinishChange( function(value) { help.refresh(); });
+
+// Temporal controller uses a folder
+var temporalFolder = datDotGUI.addFolder( "Temporal mode" );
+temporalActiveCtrl = temporalFolder.add( temporal, 'temporalIsActive'
```
Comment (jailln) [inline UDV-Core/examples/Demo.js]: I think 'var' might be needed here: var temporalActiveCtrl = ...

**Comment by jailln** [comment]
In addition I propose the following GUI-related changes: 

* The close buttons of "About" and "Help" are to small to hold their text in my navigator (Firefox 56.0; same problem on Chrome for me) (see image below). Do you have the same problem ? If yes, could you resize them please ? 

image

* I propose to put the "open controls" panel between "About" and "Help" windows to make it more accessible and so that it's not hidden when "Doc" is open.

File: UDV-Core/examples/Demo.js (hunk 1 above)
Comment (EricBoix) [reply UDV-Core/examples/Demo.js]: I added it.

**Comment by EricBoix** [comment]
The close buttons are ok for me on Firefox 57 :-(

File: UDV-Core/examples/Demo.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#datDotGUIDiv {
+    z-index: 100;
+    position: absolute;
+    left:50%;
+    top: 1px;
+    padding: 1px;
+}
+
+#datDotGUI{
+  transform:translate(-50%, 0px);
```
Comment (jailln) [inline UDV-Core/examples/Demo.css]: please replace by `transform:translate(-50%);` (I tried it and it works)

File: UDV-Core/examples/Demo.js [hunk 3]
```diff
@@ -72,8 +72,31 @@ var temporal = new udvcore.TemporalController(
...
+
+// An html container is required in order to place the dat.GUI within the page
+var datDotGUIDiv = document.createElement("div");
+datDotGUIDiv.id = 'datDotGUIDiv';
+document.body.appendChild(datDotGUIDiv);
+
+//document.getElementById("datDotGui").innerHTML = '\
+//<div id="compassWindow">\
+//<img id="compassImg" src="/img/compass.gif"></img>\
+//</div>';
```
Comment (jailln) [inline UDV-Core/examples/Demo.js]: Please remove these lines if they are not needed any more

File: UDV-Core/src/Modules/Others/MiniMap.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    border: 1px solid black;
+    background-color: white;
+    color: black;
+
+    height: 30px;
+    bottom: 0;
+    left: 5%;
+
+    pointer-events: auto
```
Comment (jailln) [inline UDV-Core/src/Modules/Others/MiniMap.css]: Missing `;`

File: UDV-Core/src/Modules/Documents/Documents.css [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+
+    border: 1px solid black;
+    background-color: white;
+    color: black;
+
+    height: 30px;
+    top: 0;
+    right: 5%;
+
+    pointer-events: auto
```
Comment (jailln) [inline UDV-Core/src/Modules/Documents/Documents.css]: Missing `;`

File: UDV-Core/src/Modules/Documents/Documents.css [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+    font-size: 14;
+
+    pointer-events: none;
+}
+
+#docBrowserPreview{
+
+    z-index: 20;
+    height: 40%;
+    display:inline-block
```
Comment (jailln) [inline UDV-Core/src/Modules/Documents/Documents.css]: Missing `;`

File: UDV-Core/src/Modules/GuidedTour/GuidedTour.css [hunk 7]
```diff
@@ -0,0 +1,308 @@
...
+
+    border: 1px solid black;
+    background-color: white;
+    color: black;
+
+    height: 30px;
+    top: 0;
+    left: 5%;
+
+    pointer-events: auto
```
Comment (jailln) [inline UDV-Core/src/Modules/GuidedTour/GuidedTour.css]: Missing `;`

**Comment by EricBoix** [comment]
We should be all set.

**Pull request merged**